packages/docs/src/locales/en/en.json

```diff
@@ -758,7 +758,8 @@
       },
       "events": {
         "clear": "Emitted if select value has been cleared",
```

Comment:
Pressing Enter should clear search text when allow-create flag is true.

![image](https://user-images.githubusercontent.com/5394573/145559004-a0132938-05f5-4d91-876a-c591cc8f549b.png)

Reply:
Fixed it